### DISCUSSION TITLE
Programmatically delete datasets from the hub

### DISCUSSION THREAD

**Discussion by nikita-galileo** [huggingface/datasets#5069] [Q&A] [answered]
Hi there! Happy to even publish the answer to this as a tool, but I'm struggling to find a way to programmatically delete datasets from the HF hub. There is `push_to_hub` but is there some equivalent of `remove_from_hub`? It the only way to remove them to manually delete them via the UI?

Thank you!

**Comment by julien-c** [comment, marked as answer]
```python
from huggingface_hub import delete_repo

delete_repo(...)
```

should work

**Reply by nikita-galileo** [reply]
Thank you!
